driverkit/utils/generate

```diff
@@ -48,8 +59,39 @@ ensure_kernelrelease() {
     TARGET_KERNEL_RELEASE="${TARGET_KERNEL%_${TARGET_KERNEL_VERSION}}"
 }
 
+function arch_to_driverkit_arch() {
+	case "$1" in
+		"x86_64")
+			echo -n "amd64"
+			;;
+		"aarch64")
+			echo -n "arm64"
```

Comment:
Are we sure @FedeDP Driverkit always looks for "arm64", even though kernel headers packages could refer to "aarch64"?

Reply:
Yep because that's the golang naming :)